test/models/assessment_test.rb

```diff
@@ -8,6 +8,28 @@ def setup; end
 
   def teardown; end
 
+  test 'update patient updated_at upon assessment create, update, and delete' do
+    patient = create(:patient)
+
+    ActiveRecord::Base.record_timestamps = false
+    patient.update(updated_at: 1.day.ago)
+    ActiveRecord::Base.record_timestamps = true
+    assessment = create(:assessment, patient: patient)
+    assert_in_delta patient.updated_at, DateTime.now, 1
```

Comment:
I think this delta is too large. It's a delta of an entire day when if anything it would be a matter of MS difference I think.

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/683/commits/dce4d4b479a47f020b4151fc583a54debffeae6e